pkg/flink/internal/store/store_test.go

```diff
@@ -38,6 +39,13 @@ func tokenRefreshFunc() error {
 	return nil
 }
 
+func (s *StoreTestSuite) TestGenerateStatementName() {
+	statementRegex := `^cli-\d{4}-\d{2}-\d{2}-\d{6}-[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$`
+	rapid.Check(s.T(), func(t *rapid.T) {
```

Comment:
I really just wanted to run the test more than once, and rapid.Check by default does 100 iterations. 
I could also just do use a loop:
```
for i := 0; i < 100; i++ {
	s.Require().Regexp(statementRegex, types.GenerateStatementName())
}
```

Reply:
Hmm, yeah they probably parallelize it too... I never really understood the benefit of fuzz testing. Why not just run the test 1 billion times before committing, instead of 100 times every time CI runs? Let's just get this merged in though. Thanks for the PR!

Reply:
You could also just run the test multiple times before committing of course, but I think that checking it in just makes it a bit more robust. If someone later on changes some behavior, CI ensures that they'll have to pass not only one but multiple randomly generated test cases.

In our particular case here though, it's probably not that helpful though I agree. The function doesn't even take any input params